requirements.txt

```diff
@@ -1,3 +1,4 @@
 spacy>=3.0,<4.0
 word2number
 en-parsigs @ https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl
+krovetzstemmer
```

Comment:
Also for CI to work, delete the `en-parsigs @ https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl` 
it is not needed there, then CI should pass